app/models/vita_partner.rb

```diff
@@ -27,15 +27,25 @@ class VitaPartner < ApplicationRecord
   has_many :intakes
   has_and_belongs_to_many :states, association_foreign_key: :state_abbreviation
   has_many :source_parameters
-  has_many :users
+
+  has_many :memberships
+  has_many :users, through: :memberships
+  accepts_nested_attributes_for :memberships
```

Comment:
This is great! Would love to see the symmetrical `has_many :vita_partners, through: :memberships`